examples/hello-world/src/main.rs

```diff
@@ -7,8 +7,9 @@ use riot_rs::debug::{exit, log::*};
 #[riot_rs::thread(autostart)]
 fn main() {
     info!(
-        "Hello from main()! Running on a {} board.",
+        "Hello from main()! Running on {} board identified as {:x}.",
```

Comment:
Please create a new example for this. We might not actually support `device_id()` on every board, limiting where `hello-world` can be built.

Reply:
Or, do we expect this to be available everywhere?

Reply:
If it can be as simple as `.unwrap_or("(a device has no name)")` and then be pretty, can it stay? (Not sure that's realistic, but we'll know that once the types are final).